driver/src/error.rs

```diff
@@ -30,7 +31,7 @@ impl From<std::io::Error> for DriverError {
 }
 impl From<web3::contract::Error> for DriverError {
     fn from(error: web3::contract::Error) -> Self {
-        DriverError::new(error.description(), ErrorKind::ContractError)
+        DriverError::new(&format!("{:}", error), ErrorKind::ContractError)
```

Comment:
This change seems to be a little bit random. Should we change all error conversions like that in a separate PR?